src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -1189,18 +1203,32 @@ export class SignAccountOpController extends EventEmitter {
 
         if (usesPaymaster) {
           try {
-            const response = await this.#callRelayer(
-              `/v2/paymaster/${this.accountOp.networkId}/sign`,
-              'POST',
-              {
+            const santinelTimeoutErr = {}
+            const paymasterTimeout = new Promise((resolve) => {
+              setTimeout(() => {
+                resolve(santinelTimeoutErr)
+              }, 8000)
```

Comment:
Wouldn't it be better if the promise is rejected with `Ambire relayer error` instead? This way:
```
   // the ugly error message below gets replaced after
            if (response === santinelTimeoutErr) throw new Error('Ambire relayer error')

            // go back to in progress after paymaster has been confirmed
            this.status = { type: SigningStatus.InProgress }
            this.emitUpdate()
```
will no longer be needed

Reply:
This is a timeout error. I don't think it will work as a promise reject

Reply:
Check this out https://github.com/AmbireTech/ambire-common/commit/365d334892d45c07bcd95e16d06af347da8e52ad

Reply:
Thanks, I will change it